src/pages/index.tsx

```diff
@@ -59,59 +60,82 @@ export default function Home() {
-                  <img src="\img\dda-bild.png" alt="Picture of DDA" />
+                {/* Top Right */}
+                <div className="lg:col-[17/33] lg:row-[1/10] lg:bg-gray/70 bg-gray max-lg:mb-24 max-lg:pt-12">
+                  <div className="lg:relative h-full">
+                    <div className="lg:absolute lg:bottom-0">
+                      <img 
+                        src="\img\dda-bild.png" 
+                        alt="Picture of DDA"
+                        className="w-full"
+                      />
+                    </div>
+                  </div>
                 </div>
-              </div>
 
-              {/* Bottom Left */}
-              <div className="md:col-[3/17] md:row-[16/25] max-md:mt-40">
-                <div className="bg-black/50 max-h-full">
-                  <img
-                    src="\img\foretagsrepresentant-bild.png"
-                    alt="Picture of people talking"
-                  />
+                {/* Bottom Left */}
+                <div className="lg:col-[3/17] lg:row-[16/25] lg:bg-gray/70 bg-gray max-lg:pt-12">
+                  <div className="lg:relative h-full">
+                    <div className="lg:absolute lg:bottom-0">      
+                      <img
+                        src="\img\foretagsrepresentant-bild.png"
+                        alt="Picture of people talking"
+                        className="w-full"
+                      />
+                    </div>
+                  </div>
                 </div>
-              </div>
 
-              {/* Bottom Right */}
-              <div className="md:col-[16/30] md:row-[18/26] text-center md:text-left z-40">
-                <div className="bg-cerise lg:p-8 p-6 drop-shadow-md overflow-hidden">
-                  <h2 className="font-black text-[#090E2F] md:text-2xl xl:text-[2rem] text-2xl pb-3">
-                    {t.home.representative}
-                  </h2>
-                  <p className="pb-8 text-[#110C30] md:text-xl xl:text-2xl text-xl">
-                    {t.home.representativeDescription}
-                  </p>
-                  <Link
-                    className="bg-white p-2.5 rounded-full text-cerise font-medium"
-                    href="/företagsanmälan"
-                  >
-                    {t.home.representativeButton}
-                  </Link>
+                {/* Bottom Right */}
+                <div className="lg:col-[16/30] lg:row-[19/26] text-center lg:text-left z-40">
+                  <div className="bg-cerise lg:p-8 p-6 drop-shadow-md overflow-hidden">
+                    <h2 className="font-black text-[#090E2F] lg:text-2xl xl:text-[1.9rem] text-2xl pb-3">
+                      {t.home.representative}
+                    </h2>
+                    <p className="pb-8 text-[#110C30] md:text-lg xl:text-2xl text-xl font-light">
+                      {t.home.representativeDescription}
+                    </p>
+                    <button 
```

Comment:
I think this should still be an `a` tag (which the `Link` is), since that allows stuff like opening in a new tab